web/webkit/src/test/scala/net/liftweb/builtin/snippet/MenuSpec.scala

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2015 WorldWide Conferencing, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.liftweb
+package builtin.snippet
+
+import xml._
+import org.specs2.matcher.XmlMatchers
+import org.specs2.mutable.Specification
+
+import common._
+import http._
+import mockweb._
+import MockWeb._
+import mocks._
+import sitemap._
+import util.Helpers.{randomString, secureXML}
+
+/**
+ * System under specification for Menu.
+ */
```

Comment:
This seems like a pretty redundant comment? Might be auto-generated though, dunno.